pytest_console_scripts/__init__.py

```diff
@@ -124,6 +127,89 @@ def print(self) -> None:
         print('# Script stderr:', self.stderr, sep='\n')
 
 
+def _handle_command_args(
+    command: str | os.PathLike[str] | Sequence[str | os.PathLike[str]],
+    *args: str | os.PathLike[str],
+    shell: bool = False,
+) -> Sequence[str | os.PathLike[str]]:
+    """Return command arguments in a consistent list format.
+
+    If shell=True then this function tries to mimic local shell execution.
+    """
+    if shell:
+        if args or not isinstance(command, (str, os.PathLike)):
+            command = subprocess.list2cmdline(
+                str(arg)
+                for arg in _handle_command_args(command, *args, shell=False)
+            )
+        command = shlex.split(str(command), posix=os.name == "posix")
```

Comment:
A formatting style comment: I was trying to use single quotes where possible in this project (basically [this rule](https://peps.python.org/pep-0008/#string-quotes) with a preference for single quotes where both would be ok). Would be nice to keep it this way.

Happy to discuss if you have another preference.

Reply:
I've fixed the quotes, including some already existing uses of double quotes discovered by a linter.

Reply:
Thanks for fixing the quotes.

I guess you have had good experience with Black if you keep using it. I looked at the way it formats when it just came out and found that in some cases the readability was not the best, so I settled for continuing with manual formatting following some style that made sense, which is not great, because it's more work and I have also noticed that my vision of the best style drifts over time.

Perhaps we could switch this project to Black formatting. It's not very big, so even if I think that readability is harmed, it won't be a big deal. At the same time it has some pretty tricky formatting situations, so it will be a good way to see what the result looks like and feel how the workflow changes.